src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -21,6 +21,7 @@ public static class ScriptConstants
         public const string HstsMiddlewareRequestDelegate = "MS_HstsMiddlewareRequestDelegate";
         public const string CorsMiddlewareRequestDelegate = "MS_CorsMiddlewareRequestDelegate";
         public const string EasyAuthMiddlewareRequestDelegate = "MS_EasyAuthMiddlewareRequestDelegate";
+        public const string AppInsightsLogPrefix = " MS_APPLICATION_INSIGHTS_LOGS";
```

Comment:
will let app insight team to update their prefix to exclude the empty char at beginning.

Reply:
Why does there have to be an empty char at the beginning?

Reply:
This is already fixed by app insight team. Before the agent log has prefix 
`LanguageWorkerConsoleLog MS_APPLICATION_INSIGHTS_LOGS...`
after talke with them the prefix is update to 
`LanguageWorkerConsoleLogMS_APPLICATION_INSIGHTS_LOGS...`, now the space is gone. I will update this prefix to `MS_APPLICATION_INSIGHTS_LOGS` in next commit.

Reply:
Ahh gotcha, I misread your comment - thanks for clarifying :)